package.json

```diff
@@ -4,13 +4,30 @@
   "description": "Handler function for AWS Lambda",
   "main": "index.js",
   "scripts": {
-    "test": "node test.js"
+    "build": "./node_modules/.bin/parcel build src/lambda.js --target=node --global handler -o lambda.js --bundle-node-modules --no-source-maps",
+    "package": "mkdir build && yarn build && zip build/deploy.zip -FSj dist/*",
```

Comment:
😵 new problem 😵: now running `yarn package` the first fresh time works (🎉) but subsequent times nets: `mkdir: build: File exists`. We probably want `mkdir -p`.

Reply:
WAY AHEAD OF YOU JUSTIN

Reply:
but r u tho